miden-lib/src/transaction/mod.rs

```diff
@@ -135,20 +129,23 @@ impl TransactionKernel {
     ///
     /// The data on the stack is expected to be arranged as follows:
     ///
-    /// Stack: [TXSR, CNC, FAH]
+    /// Stack: [CNC, FAH]
     ///
     /// Where:
-    /// - TXSR is the transaction script root.
     /// - CNC is the commitment to the notes created by the transaction.
     /// - FAH is the final account hash of the account that the transaction is being
     ///   executed against.
-    pub fn parse_output_stack(stack: &StackOutputs) -> (Digest, Digest, Digest) {
-        // TODO: use constants
-        let tx_script_root = stack.get_stack_word(0).expect("first word missing").into();
-        let output_notes_hash = stack.get_stack_word(4).expect("second word missing").into();
-        let final_account_hash = stack.get_stack_word(8).expect("third word missing").into();
-
-        (final_account_hash, output_notes_hash, tx_script_root)
+    pub fn parse_output_stack(stack: &StackOutputs) -> (Digest, Digest) {
+        let output_notes_hash = stack
+            .get_stack_word(OUTPUT_NOTES_COMMITMENT_WORD_IDX * 4)
+            .expect("second word missing")
+            .into();
+        let final_account_hash = stack
+            .get_stack_word(FINAL_ACCOUNT_HASH_WORD_IDX * 4)
+            .expect("third word missing")
```

Comment:
I think the strings in the `.expect()` calls should be updated.

Reply:
Oh, nice catch. Updated.